tests/Terra/SignerTests.cpp

```diff
@@ -17,105 +16,106 @@
 #include <gtest/gtest.h>
 #include <google/protobuf/util/json_util.h>
 
-using namespace TW;
 namespace TW::Cosmos::tests {
 
-TEST(TerraSigner, SignSendTx) {
+TEST(TerraClassicSigner, SignSendTx) {
     auto input = Proto::SigningInput();
-    input.set_signing_mode(Proto::Protobuf);
+    input.set_signing_mode(Proto::JSON);
     input.set_account_number(1037);
-    input.set_chain_id("phoenix-1");
+    input.set_chain_id("columbus-5");
     input.set_memo("");
-    input.set_sequence(1);
+    input.set_sequence(2);
```

Comment:
Are these changes intended? Look like affecting test contents.

Reply:
I separated Terra/TerraV2 tests into different files. This is just a diff being displayed inconveniently.